Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeNode.h

```diff
@@ -71,6 +71,10 @@ class RemoteLayerTreeNode : public CanMakeWeakPtr<RemoteLayerTreeNode> {
     void setActingScrollContainerID(WebCore::GraphicsLayer::PlatformLayerID value) { m_actingScrollContainerID = value; }
     void setStationaryScrollContainerIDs(Vector<WebCore::GraphicsLayer::PlatformLayerID>&& value) { m_stationaryScrollContainerIDs = WTFMove(value); }
 
+#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+    void reconcileAfterScroll();
```

Comment:
This seems currently like a very vague name that should perhaps mention interaction regions somehow?